pkg/cmd/server/origin/controller/interfaces.go

```diff
@@ -19,6 +19,10 @@ type ControllerContext struct {
 
 	DeprecatedOpenshiftInformers shared.InformerFactory
 
+	// RootClient should only be used for pre-initialization controllers
+	// (iow. serviceaccount-tokens controller)
+	RootClientBuilder *controller.SimpleControllerClientBuilder
```

Comment:
I would try to wire this down separately into the init function when you create the struct the init function lives on.  It's not a preferred means of creating the init function (makes it harder to read), but we really don't want this escaping